docs/starters.yml

```diff
@@ -1685,6 +1685,21 @@
     - Full setup instructions included
     - Documented to get you set up with Craft CMS quickly
     - Code referenced in repo
+- url: https://gatsby-starter-buttercms.netlify.com/
+  repo: https://github.com/ButterCMS/gatsby-starter-buttercms
+  description: A starter template for spinning up a Gatsby+ ButterCMS site
+  tags:
+    - Blog
+    - FAQ
```

Comment:
```suggestion
```

We don't have a FAQ tag yet and it's not really something we want to add as a tag